src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp

```diff
@@ -0,0 +1,79 @@
+#include <ios>
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <regex>
+#include "rsyslog_plugin.h"
+#include "common/logger.h"
+#include "common/json.hpp"
+#include "common/events.h"
+
+using namespace std;
+using namespace swss;
+using json = nlohmann::json;
+
+void RsyslogPlugin::onMessage(string msg) {
+    string tag = "";
+    event_params_t param_dict;
+    if(!parser->parseMessage(msg, tag, param_dict)) {
+        SWSS_LOG_INFO("%s was not able to be parsed into a structured event\n", msg.c_str());
```

Comment:
This code line is hit, for every log line that is not in regex. Correct?
More like most common path, as most of the logs will not be matched.

If so, we can't log it as info. May be debug.

Reply:
Done